### PR TITLE
Hide links in footer when map is active, set tabindex leaflet link, a…

### PR DESCRIPTION
**Pull request by OscarBakker**
…djust tests.

punten van herjen verwerkt!

https://datapunt.atlassian.net/browse/SIG-4477?atlOrigin=eyJpIjoiNzVmMDQ0NDE1N2MxNDc2MzllZTJhYjEwZWQ3MzU4YTIiLCJwIjoiaiJ9

## Signalen

Before opening a pull request, please ensure:

- [x] Double-check your branch is based on `develop` and targets `develop`
- [x] Pull request has tests (we are going for 100% coverage!)
- [x] Code is well-commented, linted and follows project conventions
- [x] Committed source code is headed by the correct SPDX license expression

Be kind to code reviewers, please try to keep pull requests as small and focused as possible :)

### CODE REVIEW

File: src/signals/incident/components/form/MapSelectors/Asset/AssetSelect.test.tsx [hunk 1]
```diff
@@ -5,19 +5,19 @@ import { render, screen } from '@testing-library/react'
...
 import type { Location } from 'types/incident'
 import { Provider } from 'react-redux'
-import { setupStore } from 'signals/incident/containers/IncidentContainer/testReducer'
 import {
   UNKNOWN_TYPE,
   UNREGISTERED_TYPE as mockUNREGISTERED_TYPE,
   UNREGISTERED_TYPE,
 } from '../constants'
 import type { Item } from '../types'
+import configureStore from '../../../../../../configureStore'
```
Comment (kiruys) [inline src/signals/incident/components/form/MapSelectors/Asset/AssetSelect.test.tsx:20]: daar ben ik weer 🤓 , kun je het pad specificeren?

File: src/components/FooterContainer/index.tsx [hunk 2]
```diff
@@ -4,6 +4,8 @@ import { Link, Footer, FooterTop, FooterBottom } from '@amsterdam/asc-ui'
 import styled from 'styled-components'
 
 import configuration from 'shared/services/configuration/configuration'
+import { useSelector } from 'react-redux'
+import { makeSelectIncidentContainer } from '../../signals/incident/containers/IncidentContainer/selectors'
```
Comment (kiruys) [inline src/components/FooterContainer/index.tsx:8]: volgens mij kunnen de eerste twee '../../' weg

File: src/components/Map/Map.tsx [hunk 3]
```diff
@@ -70,14 +70,33 @@ const Map: FC<PropsWithChildren<MapProps>> = ({
...
+      // @ts-ignore
+      const leafletControlAttribution: HTMLLinkElement = document.querySelector(
+        '.leaflet-control-attribution a'
+      )
+
+      if (leafletControlAttribution) {
+        leafletControlAttribution.tabIndex = -1
+      }
+    }, 0)
+
```
Comment (kiruys) [inline src/components/Map/Map.tsx:97]: nogal hacky, maar blij dat het gelukt is

**Review by kiruys — COMMENTED** [review]
Fijn dat het gelukt is, Oscar!
Zou je ook nog even die non-null assertion warning kunnen fixen?

File: src/components/Map/Map.tsx (hunk 3 above)
Comment (OscarBakker) [reply src/components/Map/Map.tsx:97]: haha ja echt heh

**Pull request merged**